__tests__/app.test.js

```diff
@@ -156,3 +156,56 @@ describe("GET /api/articles/:article_id/comments", () => {
       })
   });
 });
+
+describe("POST /api/articles/:article_id", () => {
+  it("201: response with the created comment", () => {
+    const input = { username: "rogersop", body: "hEllo wOrld!" };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(input)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.created).toMatchObject({
+          comment_id: 19,
+          body: 'hEllo wOrld!',
+          article_id: 2,
+          author: 'rogersop',
+          votes: 0,
+          created_at: expect.any(String)        
+        });
+      });
+  });
+
+  it("400: bad article id", () => {
+    const input = { username: "rogersop", body: "hEllo wOrld!" };
+    return request(app)
+    .post("/api/articles/notAnId/comments")
+    .send(input)
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe('bad identity format');
+    });
+  });
+
+  it("404: article not found", () => {
+    const input = { username: "rogersop", body: "hEllo wOrld!" };
+    return request(app)
+      .post('/api/articles/999/comments')
+      .send(input)
+      .expect(404)
+      .then(({ body }) => {
+          expect(body.msg).toBe('article not found');
+      })
+  });
+
+  it("400: bad username or user does not exist", () => {
+    const input = { username: "rogerfederer", body: "hEllo wOrld!" };
+    return request(app)
+      .post('/api/articles/2/comments')
+      .send(input)
+      .expect(400)
+      .then(({ body }) => {
+          expect(body.msg).toBe('foreign key error');
+      })
```

Comment:
Just one more test please

- [ ] 400 missing required field in the post body object